### ISSUE TITLE
Dashboard Header - Settings icon - Lack of Descriptive Labels for Buttons

### ISSUE THREAD

**Issue by Pushpajyothi** [awslabs/iot-app-kit#2040]
Description : 
For "Cancel" and "Confirm" lack clear labels, which may result in confusion for all users, particularly those relying on assistive technologies.  
Aria-label attribute is missing for the buttons.

Image Link : 
https://imgur.com/3Qkz8RV
